contracts/contribute/ContributeManager.sol

```diff
@@ -39,6 +39,12 @@ error WrongStakingInstance(address stakingInstance);
 /// @param multisig Multisig address.
 error WrongServiceSetup(uint256 socialId, uint256 serviceId, address multisig);
 
+/// @dev Wrong service state.
+/// @param socialId Social Id.
+/// @param serviceId Service Id.
+/// @param state Service state.
+error WrongServiceState(uint256 socialId, uint256 serviceId, uint8 state);
```

Comment:
Maybe make sense to kick out `socialId` from everywhere, can be always fetched from the `Contributors` contract mappings.